docs/data/tree-view/overview/overview.md

```diff
@@ -75,7 +75,29 @@ Or increasing the width of the item state indicator to be full-width:
 
 ## Customization
 
-### Custom icons, border and animation
+### Custom icons
+
+You can customize the collapse, expand and end icons of the TreeView.
+
+You can use icon liraries:
+
+{{"demo": "CustomMaterialIcons.js", "defaultCodeOpen": true}}
+
+Or you can create your own components:
+
+{{"demo": "CustomIcons.js", "defaultCodeOpen": false}}
```

Comment:
As much as I love how this one looks, I wonder if we need it? 😕 Maybe we can consolidate this with the above and have one prop use an icon from a library and another prop to use a custom "locally"-created icon, with the `SvgIcon` component?